nsxt/policy_common.go

```diff
@@ -437,7 +437,10 @@ func getPolicyRulesFromSchema(d *schema.ResourceData) []model.Rule {
 			Services:             getPathListFromMap(data, "services"),
 			Scope:                getPathListFromMap(data, "scope"),
 			Profiles:             getPathListFromMap(data, "profiles"),
-			SequenceNumber:       &sequenceNumber,
+		}
+
+		if sequenceNumber > 0 {
+			elem.SequenceNumber = &sequenceNumber
```

Comment:
So what would be the value of elem.SequenceNumber if sequenceNumber == 0? Wouldn't it be the same?
Should this just protect from negative values in sequenceNumber?

Reply:
The value will be nil, which translates to not passing the value in request body. In this case NSX assigns the value, which is what we want.